### PR TITLE
Initial Berkeley task monitor

### PR DESCRIPTION
**Pull request by hdefazio**

### CODE REVIEW

**Comment by hdefazio** [comment]
Sorry for the file dump  - Summary: 

- Task monitor:
ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py

- Relevant Berkeley files:
angel_system/berkeley/demo/predictor.py
angel_system/berkeley/detectron2/utils/tracker.py
angel_system/berkeley/detectron2/utils/visualizer.py

- Berkeley's model was added to the ansible config

- Poetry env updates based on the detectron2 dependencies

File: ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        """
+        #import time
+        #time.sleep(2)
+        log = self.get_logger()
+        self.idx += 1
+
+        # Convert ROS img msg to CV2 image
+        bgr_image = BRIDGE.imgmsg_to_cv2(image, desired_encoding="bgr8")
+
+        predictions, step_infos, visualized_output = self.demo.run_on_image_smoothing_v2(
```
Comment (Purg) [inline ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py:190]: Should we be passing `draw_output=False` when `self._draw_output == False`?

Comment (hdefazio) [reply ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py:190]: sorry, where? self._draw_output is passed to `demo` when it is initialized

File: pyproject.toml [hunk 2]
```diff
@@ -34,12 +34,20 @@ transitions = "^0.8.11"
 trimesh = {extras = ["all"], version = "^3.10.2"}
 # For UHO Activity Classifier
 # - see arisia: /data/dawei.du/ptg-activity-recognition/activity_hydra/requirements.txt
-hydra-core = "^1.2.0"
+hydra-core = "==1.1.0"
```
[the remark this answers is not in the document]
Comment (hdefazio) [reply pyproject.toml:37]: Yeah, I think it has to be pinned. Poetry was having issues finding a version that works when I tried `>=`

File: ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py (hunk 1 above)
Comment (Purg) [reply ros/angel_system_nodes/angel_system_nodes/berkeley_task_monitor.py:194]: I see it now, I don't remember how I missed that. Resolved!

**Pull request merged**